src/schema.rs

```diff
@@ -169,6 +169,7 @@ pub(crate) struct Team {
     website: Option<WebsiteData>,
     #[serde(default)]
     lists: Vec<TeamList>,
+    discord_role: Option<DiscordRole>,
```

Comment:
I would have a `Vec<DiscordRole>` instead of `Option<DiscordRole>` just in case.